worlds/ff6wc/WorldsCollide/log/format.py

```diff
@@ -53,5 +53,11 @@ def get_separator(label):
 
     return separator
 
-def format_option(option, value):
+def format_option(option, value, unique_name = ''):
+    from constants.standard_flags import standard_flags
```

Comment:
Does this need the relative import?

Reply:
Yes that most likely does need to be updated; I think this code is currently stubbed out in the base FF6WC, so that's why I haven't gotten any errors when running across it yet in my testing.